docs/whats-new/production-release-notes.md

```diff
@@ -1,5 +1,17 @@
 # Production release notes
 
+## April 2025
+
+**16 April 2025**
+
+Frontend version: 1.0.0-20250404.1440 | Backend version:20250407.1505-1.156.0
+
+| Type   | Change | Description |
+|---|---|---|
+| **Change** | `Invite Public Officer Automation API` – `mobileNumber` is now required | The `mobileNumber` field is now mandatory when calling the `InvitePublicOfficer` API, except for private route usage (such as automatic invitations via the `sync-wog-to-techpass-group` job). |
```

Comment:
can we remove the "except for private route usage (such as automatic invitations via the `sync-wog-to-techpass-group` job)" ?